spec/inspect_spec.lua

```diff
@@ -411,4 +411,15 @@ describe( 'inspect', function()
       end)
     end)
   end)
+
+  it('allows changing the global tostring', function()
+    local save = _G.tostring
+    _G.tostring = function(x) 
```

Comment:
Why do you think this is not correct? There was a change between 5.1 and 5.2 when _ENV was introduced, but there was no change with _G. Anyway, I removed all _Gs from the inspect module, the one remaining is only in the test, which will not go into user code. For me, the _G expresses my intentions clearer, but that is personal taste. The whole thing with redefining tostring is brittle anyway. It works well in my embedded system, but fails in ZeroBrane Studio. But the changes here do no harm.